rescrape.py

```diff
@@ -203,38 +237,51 @@
 # search results from the original Zillow search web page. If they are not equal, it may mean more or less sold home
 # URLs have been scraped than what the original Zillow search result count was from the original Zillow web page.
 if len(SoldHomeZillowLinks) == ZillowResultCount:
-    print('The number of Sold Home Zillow Links scraped from all pages equals \n'
-          'the Zillow Result Count that was scraped from the initial page.')
+    print(
+        "The number of Sold Home Zillow Links scraped from all pages equals \n"
+        "the Zillow Result Count that was scraped from the initial page."
+    )
 else:
-    print("WARNING!!! The number of Sold Home Zillow Links scraped from all pages DOES NOT EQUAL \n"
-          "the Zillow Result Count that was scraped from the initial page. There may be an issue with Zillow \n"
-          "recommending/providing additional links on subsequent pages that weren't originally included in \n"
-          "the Zillow Result Count.")
+    print(
+        "WARNING!!! The number of Sold Home Zillow Links scraped from all pages DOES NOT EQUAL \n"
+        "the Zillow Result Count that was scraped from the initial page. There may be an issue with Zillow \n"
+        "recommending/providing additional links on subsequent pages that weren't originally included in \n"
+        "the Zillow Result Count."
+    )
 
 # Loop over each recently sold home Zillow URL and scrape pertinent sold home details and information, and print the
 # results to the screen.
-print('{:^55s}{:^10s}{:>10s}{:>6s}{:^17s}{:^7s}{:^22s}{:^9s}{:^17s}{:^12s}'.format(
-        'Address', 'Sell Price', 'Bed/Bath', 'SqFt', 'Type', 'Built', 'Heating', 'Cooling', 'Parking', 'Lot Size'))
+print(
+    "{:^55s}{:^10s}{:>10s}{:>6s}{:^17s}{:^7s}{:^22s}{:^9s}{:^17s}{:^12s}".format(
+        "Address",
+        "Sell Price",
+        "Bed/Bath",
+        "SqFt",
+        "Type",
+        "Built",
+        "Heating",
+        "Cooling",
+        "Parking",
+        "Lot Size",
+    )
+)
 
 for i in range(0, len(SoldHomeZillowLinks)):
 
-    # A delay of 3 seconds between each request is instituted to ensure the Zillow servers are not overwhelmed.

```

Comment:
As mentioned above, we can get rid of this right now. But note that if we start seeing HTTP request issues, this may be a good place to start for troubleshooting.